editor/src/templates/editor-canvas.tsx

```diff
@@ -357,31 +322,6 @@ function on(
     } else {
       return [CanvasActions.scrollCanvas(event.delta as any as CanvasVector)]
     }
-  } else if (
```

Comment:
Is it maybe worth seeing if we can adapt this to use the canvas strategies work so that we can restore the canvas scrolling when dragging past the bounds of the canvas?

Reply:
(I guess as a follow up PR?)

Reply:
added a TODO in the paper doc